src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -409,6 +405,18 @@ internal Accessibility LocalAccessibility
             get { return ModifierUtils.EffectiveAccessibility(this.DeclarationModifiers); }
         }
 
+        /// <summary>
+        /// Indicates whether this accessor itself has a 'readonly' modifier.
+        /// </summary>
+        internal bool LocalDeclaredReadOnly => (DeclarationModifiers & DeclarationModifiers.ReadOnly) != 0;
```

Comment:
Prefix `bool` based properties with Is, Has, etc ...

Reply:
Rather than local consider using "directly". It would be easy for local to get confused with locals. I think "direct" is harder to mis-interpret here. 

---
In reply to: [268237180](https://github.com/dotnet/roslyn/pull/34338#discussion_r268237180) [](ancestors = 268237180)

Reply:
The idea was to make it resemble the property LocalAccessibility which indicates any accessibility mods directly on the accessor.

Reply:
Okay. That makes sense. I'll take prior art over 8AM opinions.

Reply:
I’ll add an “Is” though to better suit the bool return type.